test/unit/mdc-switch/foundation.test.js

```diff
@@ -106,3 +128,21 @@ test('#handleChange removes mdc-switch--checked from the switch when it is an un
   foundation.handleChange({target: {checked: false}});
   td.verify(mockAdapter.removeClass(MDCSwitchFoundation.cssClasses.CHECKED));
 });
+
+test('#handleChange sets aria-checked to true when the swith is a checked state', () => {
```

Comment:
nit: typo

Reply:
Thank you!

Fixed `swith` -> `switch`.